config/api.yml

```diff
@@ -740,6 +740,7 @@
     :options:
     - :collection
     - :custom_actions
+    - :subcollection
```

Comment:
With this exposed as a subcollection, aren't these now fetchable ?  i.e. GET /api/container_nodes/:id/container_images ?  If so we'd need specs to test those happy (and unhappy / i.e. invalid role) paths.

Reply:
This means it will be available for any model that `has_many :container_images`  ? If so it will also be there for Container Groups/Projects. If so I will make tests for these cases as well, but seems like there should be a generic way to test it.

Reply:
I get 404 for all these cases so I guess it is not possible to access them this way.

Reply:
Could you add tests for GET /api/providers/:id/container_images and GET /api/providers/:id/container_images/:container_image_id ?